dashboard/config/linux/android-5.15.config

```diff
@@ -1,8 +1,11 @@
+# Automatically generated by syz-kconf; DO NOT EDIT.
+# Kernel: https://android.googlesource.com/kernel/common 889a0c39fef31
+
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 5.15.64 Kernel Configuration
+# Linux/x86_64 5.15.64 Kernel Configuration
 #
-CONFIG_CC_VERSION_TEXT="clang version 14.0.5"
+CONFIG_CC_VERSION_TEXT="clang version 14.0.5 (Fedora 14.0.5-1.fc36)"
```

Comment:
Did you use syz-env, as recommended in https://github.com/google/syzkaller/blob/master/dashboard/config/linux/README.md ?

`syz-env make configs SOURCEDIR=/path/to/existing/linux/checkout`